packages/account/src/tests/test_utils.cairo

```diff
@@ -0,0 +1,58 @@
+use openzeppelin_testing::constants::{TRANSACTION_HASH, stark};
+use openzeppelin_testing::signing::SerializedSigning;
+use snforge_std::{ContractClassTrait, DeclareResultTrait, declare};
+use crate::utils::assert_valid_signature;
+
+//
+// assert_valid_signature
+//
+
+#[test]
+fn test_assert_valid_signature_valid() {
+    let key_pair = stark::KEY_PAIR();
+    let contract_class = declare("DualCaseAccountMock").unwrap().contract_class();
+    let mut calldata = array![];
+    key_pair.public_key.serialize(ref calldata);
+    let (account_address, _) = contract_class.deploy(@calldata).unwrap();
```

Comment:
For consistency, let's use the `openzeppelin_testing::declare_and_deploy` function, and move this setup step into a separate function called setup.